client/src/components/utils/appHelper.js

```diff
@@ -176,7 +253,10 @@ const useCurrentLocationClickHandler = (props, interactionTypes, dispatchFunctio
   props.changeInteractionType(interactionTypes.SELECTING_ROUTE);
 };
 
-
+/**
+ * Cancel request to get route
```

Comment:
"Cancel route currently displayed"

This doesn't actually cancel a **_request_**; the request has already been made and succeeded. This simply removes the route from the map by resetting the redux store to the default settings.